x-pack/plugins/runtime_fields/README.md

```diff
@@ -0,0 +1,136 @@
+# Runtime fields
+
+Welcome to the home of the runtime field editor and everything related to runtime fields!
+
+## The runtime field editor
+
+The runtime field editor is exported in 2 flavours:
+
+* As the content of a `<EuiFlyout />`
+* As a standalone component that you can inline anywhere
+
+### Content of a `<EuiFlyout />`
+
+```js
+import React, { useState } from 'react';
+import { EuiFlyoutBody, EuiButton } from '@elastic/eui';
+import { RuntimeFieldEditorFlyout, RuntimeField } from '../runtime_fields/public';
+
+const MyComponent = () => {
+  const { docLinksStart } = useCoreContext(); // access the core start service
+  const [isFlyoutVisilbe, setIsFlyoutVisible] = useState(false);
+ 
+  const saveRuntimeField = useCallback((field: RuntimeField) => {
+    // Do something with the field
+  }, []);
+
+  return (
+    <>
+      <EuiButton onClick={() => setIsFlyoutVisible(true)}>Create field</EuiButton>
+
+      {isFlyoutVisible && (
+        <EuiFlyout onClose={() => setIsFlyoutVisible(false)}>
```

Comment:
I think I recall us talking about something similar to this in the past but why is the `RuntimeFieldEditorFlyout` wrapped in `EuiFlyout`?

Reply:
So it can be injected in a flyout when calling `core.overalys.openFlyout()`. It is a similar mechanism that I put in place for the `<GlobalFlyout />`.

Reply:
Good idea, I will rename the component 👍